database/buildSchema/37_application_list_view.sql

```diff
@@ -15,13 +19,15 @@ CREATE OR REPLACE VIEW "public".application_list AS
 	-- 	template_questions_count(app),
 	-- 	assigned_questions_count(app, stage_status.stage_id, 1),
 		assigned_questions_count(app, stage_status.stage_id, 1) >= template_questions_count(app)
-		AS is_fully_assigned_level_1
+		AS is_fully_assigned_level_1,
```

Comment:
Not from this PR, sorry I just have investigated if this `is_fully_assigned` would return what is expected, but I don't this it is considering questions that aren't visible therefore not assigned in the review... I think I'll create a separated issue for that. Just pointing out here so we can discuss - maybe I'm just not seeing something

Reply:
I'd have thought that when you assign a Section (or a whole application), they'd be assigned regardless of whether or not question is visible. In fact, back-end has no concept of "is_visible"  as it's only ever calculated on-the-fly in the front end (and changes as responses change). You assign reviews based on template_elements, don't you, rather than specific applications responses? If not, they probably should be. You don't want a situation where a reviewer is assigned a section, and then the applicant makes changes which makes another question visible but no one is assigned to it. Surely we assigned a selection of "template elements" which are tied to responses, not responses themselves?